Adyen/UI/List/ListItem.swift

```diff
@@ -46,14 +49,16 @@ public class ListItem: FormItem {
     ///   - style: The list item style.
     ///   - identifier: The `accessibilityIdentifier` to be used on the `ListItem`
     ///   - selectionHandler: The closure to execute when an item is selected.
+    ///   - accessibilityLabel: An optional custom `accessibilityLabel` to use. Defaults to title + subtitle + trailingText joined by a `,`
     public init(
         title: String,
         subtitle: String? = nil,
         icon: Icon? = nil,
         trailingText: String? = nil,
         style: ListItemStyle = ListItemStyle(),
         identifier: String? = nil,
-        selectionHandler: (() -> Void)? = nil
+        selectionHandler: (() -> Void)? = nil,
+        accessibilityLabel: String? = nil
```

Comment:
isn't it better to keep the closure at the end as trailing?

Reply:
yes, good point! will change 👌